src/components/authorized/MyHomePage.js

```diff
@@ -6,32 +6,25 @@ import axios from 'axios';
 import Navbar from '../Navbar';
 import UserVerificationButton from '../UserVerificationButton';
 
+import CustomerCommissionList from '../shared/uicomponents/CustomerCommissionList';
+
 export default class MyHomePage extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
             loading: true,
             errorMessage: '',
-            customerCommissions: [],
+            userId: '',
```

Comment:
can you find out if userId was returned as an int or a string?

Reply:
userId is an int. would 0 be a better initial value?

Reply:
nevermind, by fixing the problem below, i no longer have to initialize the userId

Reply:
for best practices you could probably initialize as `null`